logstash-core/lib/logstash/config/pipelines_info.rb

```diff
@@ -93,11 +99,19 @@ def self.format_pipeline_vertex_section_stats(stats, pipeline)
 
           acc
         end
-        
-        acc << {
+        if LogStash::PluginMetadata.exists?(plugin_id.to_s)
+          cluster_uuids = LogStash::PluginMetadata.for_plugin(plugin_id.to_s).get(:cluster_uuid)
```

Comment:
nitpick: since this now holds at most one uuid, the variable name should be singular `cluster_uuid`.

Reply:
Good catch. Thanks! Fixed.